assets-api.md

```diff
@@ -410,3 +410,93 @@ Retrieve assets cost data for the past seven days, aggregated by type, and as cu
 ```
 {% endtab %}
 {% endtabs %}
+ 
+ Retrieve all AWS S3 assets cost data for the past seven days.
+ 
+ {% tabs %}
+{% tab title="Request" %}
+`http://localhost:9090/model/assets?window=7d&filterServices=AmazonS3"`
+{% endtab %}
+
+{% tab title="Response" %}
+```json
+{
+  "code": 200,
+  "data": [
+    {
+       "AWS/297945954695/__undefined__/Storage/__undefined__/Cloud/AmazonS3/cloud-bench-1/__undefined__": {
+        "type": "Cloud",
+        "properties": {
+          "category": "Storage",
+          "provider": "AWS",
+          "account": "297945954695",
```

Comment:
I suggest we obscure the account IDs in the response.

Reply:
Good call! I have removed them.